app/Http/Controllers/BookController.php

```diff
@@ -7,10 +7,48 @@
 
 class BookController extends Controller
 {
-    public function getAll()
+    public function getAll(Request $request)
     {
+        // Validate and throw 422 errors
+        $request->validate([
+            'claimed' => 'nullable|numeric',
+        ]);
+
+        $hidden =
+            [
+                'genre_id',
+                'created_at',
+                'updated_at',
+                'blurb',
+                'page_count',
+                'year',
+                'claimed_by_name',
+                'claimed_by_email',
+                'claimed'
+            ];
+
+        // If no param provided, get Unclaimed books
+        $claimed = $request->query('claimed', 0);
```

Comment:
I've just noticed that there is a discrepency between the story and documentation. The story says to get unclaimed books by default, but the documentation says that the route should get all books by default, and claimed with 1, unclaimed with 0. 

In this case, I think it's best to go with what the documenation says. Make it so this route gets all books by default, and only filters if claimed is 1 or 0